### PR TITLE
fix(RDBMS customisation): Add new grant required by MySQL 8.0

### PR DESCRIPTION
**Pull request by ugaston**
Version: 7.9

Add the new grant required by MySQL 8.0 to support XA transactions.

### CODE REVIEW

File: md/database-configuration.md [hunk 1]
```diff
@@ -237,6 +237,16 @@ See [in-doubt xact resolution Server Configuration Option](https://msdn.microsof
 
 #### MySQL
 
+##### XA Transactions
+
+To support XA transactions, starting from MySQL 8.0, special XA rights must be granted to the database user. For example, if the users for the engine and BDM databases are `bonita` and `business_data` respectively, you should run the following command:
+
+```sql
+GRANT XA_RECOVER_ADMIN ON *.* to bonita, business_data;
```
Comment (tbouffard) [inline md/database-configuration.md:245]: ⚠️ we may add warnings about this because this grant is for all objects.
Could we restrict that?

Comment (ugaston) [reply md/database-configuration.md:245]: Good point.
I added a warning message. (See latest commit.)

Comment (tbouffard) [reply md/database-configuration.md:245]: 👍

File: md/database-configuration.md [hunk 2]
```diff
@@ -237,6 +237,22 @@ See [in-doubt xact resolution Server Configuration Option](https://msdn.microsof
...
+
+FLUSH PRIVILEGES;
+```
+
+::: warning
+Please note that the above command, by specifying **`*.*`**, is granting the right for all the databases. You may want to limit the command to your engine and BDM databases if there are additional databases in your MySQL server.
```
Comment (tbouffard) [inline md/database-configuration.md]:
```suggestion
Please note that the above command, by specifying **`*.*`**, is granting privileges for all databases. You may want to limit them to the Bonita BPM and BDM databases if there are additional databases in your MySQL server.
```

File: md/database-configuration.md [hunk 3]
```diff
@@ -237,6 +237,22 @@ See [in-doubt xact resolution Server Configuration Option](https://msdn.microsof
 
 #### MySQL
 
+##### XA Transactions
+
+To support XA transactions, starting from MySQL 8.0, special XA rights must be granted to the database user. For example, if the users for the engine and BDM databases are `bonita` and `business_data` respectively, you should run the following command:
+
+```sql
+GRANT XA_RECOVER_ADMIN ON *.* to bonita, business_data;
```
Comment (educhastenier) [inline md/database-configuration.md:245]: I don't understand... I am convinced I *did* this contribution already, when I discovered this new grant requirement for 7.9 .... :thinking: ...

File: md/database-configuration.md [hunk 4]
```diff
@@ -237,6 +237,22 @@ See [in-doubt xact resolution Server Configuration Option](https://msdn.microsof
 
 #### MySQL
 
+##### XA Transactions
+
+To support XA transactions, starting from MySQL 8.0, special XA rights must be granted to the database user. For example, if the users for the engine and BDM databases are `bonita` and `business_data` respectively, you should run the following command:
```
Comment (ugaston) [inline md/database-configuration.md]:
```suggestion
To support XA transactions, starting from MySQL 8.0, special XA rights must be granted to the database user. For example, if the users for the Bonita BPM and BDM databases are `bonita` and `business_data` respectively, you should run the following command:
```

**Review by tbouffard — APPROVED** [review]
LGTM
@ugaston feel free to merge if ok for you

**Pull request merged**